[0;1;31mWarning:[0m The unit file, source configuration file or drop-ins of daphne.service changed on disk. Run 'systemctl daemon-reload' to reload units.
[0;1;32m‚óè[0m daphne.service - WebSocket Daphne Service
     Loaded: loaded (/etc/systemd/system/daphne.service; disabled; vendor preset: enabled)
     Active: [0;1;32mactive (running)[0m since Sat 2023-02-18 19:18:51 UTC; 12h ago
   Main PID: 93680 (python)
      Tasks: 4 (limit: 2351)
     Memory: 56.4M
     CGroup: /system.slice/daphne.service
             ‚îî‚îÄ93680 /home/ubuntu/advertising-tv-control/env/bin/python /home/ubuntu/advertising-tv-control/env/bin/daphne -b 0.0.0.0 -p 8001 server.asgi:application

Feb 19 08:09:14 ip-172-31-1-223 python[93680]:     self.receive(text_data=message["text"])
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:   File "/home/ubuntu/advertising-tv-control/server/./pi/consumers.py", line 725, in receive
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:     text_data_json = json.loads(text_data)
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:   File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:     return _default_decoder.decode(s)
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:   File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:   File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
Feb 19 08:09:14 ip-172-31-1-223 python[93680]:     raise JSONDecodeError("Expecting value", s, err.value) from None
Feb 19 08:09:14 ip-172-31-1-223 python[93680]: json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
